### DISCUSSION TITLE
mini.surround plugin mappings

### DISCUSSION THREAD

**Discussion by pannet1** [LazyVim/LazyVim#1145] [Q&A]
EDIT: Sorry could be some misconfiguration issue. Now it is working fine. Please ignore this question.

lazy recommended setting for mini.surround plugin did not work.

```
return {
  "echasnovski/mini.surround",
  version = "*",
  opts = {
    mappings = {
      add = "gsa",
      delete = "gsd",
      find = "gsf",
      find_left = "gsF",
      highlight = "gsh",
      replace = "gsr",
      update_n_lines = "gsn",
    },
  },
}
```

this is not working for me either. then i found out that: instead of all the `gs` actually `gz` is working. so for example  `replace` is now mapped to `gzr`. is this the expected behaviour.

**Comment by dpetka2001** [comment]
Yes, it is expected behavior as you can see in the [docs](https://www.lazyvim.org/plugins/coding#minisurround). There is also a [recipe](https://www.lazyvim.org/configuration/recipes#change-surround-mappings) if you want to change them to whatever you want.

**Reply by folke** [reply]
Maybe I should change it to `gs` for everyone now, since leap is no longer the default, so no collision on that mapping

**Reply by dpetka2001** [reply]
Doesn't flash also use `s`?

**Reply by folke** [reply]
yes, but not `gs`

**Reply by pannet1** [reply]
i think the mini-surround mappings are prefixed with `g` to avoid the collision.  is there a possibility to replace `gs` with something else

**Reply by folke** [reply]
I chose `gz` because `gs` was used by `leap.nvim`, but since `flash.nvim` is now the default, `gs` can be used instead of `gz`.

**Comment by nipuna-perera** [comment]
This would be great! I like the 'gs' mapping much better

**Comment by pannet1** [comment]
> I chose `gz` because `gs` was used by `leap.nvim`, but since `flash.nvim` is now the default, `gs` can be used instead of `gz`.


oh i see. i must be very naive to imagine a mapping with few keystrokes .. something like `fr` for `replace` and `fa` for `add`.

thank you @folke again for creating us a wonderful neovim experience.
